mischief.js

```diff
@@ -33,19 +35,18 @@ function transferMoney(fromAccount, toAccount, amount, description) {
 	if (getAccountBalance(fromAccount) > 10000) {
 		debitAccount(fromAccount, 0.1, 'Transaction Fee') && creditAccount(account3, 0.1, 'Transaction Fee');
 	}
-	return;
 }
 
-account1 = [];
-account2 = [];
-account3 = []; // rogue developer account
+let account1 = [];
+let account2 = [];
+let account3 = []; // rogue developer account
 
 console.log("\nExpect Account1 Opening Balance       "+getAccountBalance(account1) +" === 0        \u2713 ");
-creditAccount(account1, 20000, 'Add Funds');
-transferMoney(account1,account2, 100, 'Give money to friend')
-transferMoney(account2,account1, 10, 'Transfer back 10')
+creditAccount(account1, 5000, 'Add Funds');
```

Comment:
do we _need_ to change this from `20000` to `5000` ?

Reply:
@nelsonic - It has to be an input less than 10000 to show the 96.3% for the first time we run the coverage report. Please let me know.

Reply:
@durja you're right. well-spotted. thanks for opening this PR! 👍